client/src/bin/greengrass-cli.cmd

```diff
@@ -0,0 +1,9 @@
+@echo off
+
+set SCRIPT_PATH=%~dp0
+set CLI_HOME=%SCRIPT_PATH%..
```

Comment:
I'm not familiar with the syntax. Should it be `%SCRIPT_PATH%\..`?

Reply:
`%~dp0` includes the ending `\`. So there is no need for a `\`. But good eyes!

I don't know how to make `%~dp0` not to include the ending `\`. But if possible, I agree that it would be easier to read.